app/services/library_website_search_service.rb

```diff
@@ -0,0 +1,57 @@
+class LibraryWebsiteSearchService < AbstractSearchService
+  def initialize(options = {})
+    options[:query_url] ||= Settings.LIBRARY_WEBSITE_QUERY_API_URL.to_s
+    options[:response_class] ||= Response
+    super
+  end
+
+  class Response < AbstractSearchService::Response
+    HIGHLIGHTED_FACET_FIELD = 'document_type_facet'.freeze
+    QUERY_URL = Settings.LIBRARY_WEBSITE_QUERY_API_URL.freeze
+
+    def total
+      999_999_999 # TODO: library web scrape doesn't provide the total number of hits
```

Comment:
@jvine what would you like us to do here?  I feel like we'll never get the total # of results w/ the screen scraping method.  Should this just not show any number?

Reply:
I've added afd5ee8 which does not show a number (e.g., "See all Library website results")

Reply:
@drh-stanford that's fine for now - frankly at the moment the Drupal bento can't get the total number of Drupal website results either.